model/riscv_insts_zvkned.sail

```diff
@@ -0,0 +1,342 @@
+/*=======================================================================================*/
+/*  This Sail RISC-V architecture model, comprising all files and                        */
+/*  directories except where otherwise noted is subject the BSD                          */
+/*  two-clause license in the LICENSE file.                                              */
+/*                                                                                       */
+/*  SPDX-License-Identifier: BSD-2-Clause                                                */
+/*=======================================================================================*/
+
+function clause currentlyEnabled(Ext_Zvkned) = hartSupports(Ext_Zvkned) & currentlyEnabled(Ext_V)
+
+union clause ast = ZVKVAESDFTYPE : (zvkfunct6, vregidx, vregidx)
```

Comment:
```suggestion
union clause ast = VAESDF : (zvkfunct6, vregidx, vregidx)
```
Elsewhere in the PR there are ast nodes like `VAESKF1_VI`, `VAESZ_VS` and similar, which are easier to parse; this will make it more consistent.